Symfony/CS/Tests/Fixer/Contrib/MultipleUnsetFixerTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author SpacePossum
+ */
+class MultipleUnsetFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideCases()
+    {
+        return array(
+            array(
+                '<?php //1
+                    unset($foo/*;*/,/*;*/$bar, $c ,$foobar  ,  $foobar2);
+                     //test
+                     /* more comment test*/
+                    '.'
+                ',
+                '<?php //1
+                    unset($foo/*;*/);
+                    unset(/*;*/$bar, $c ); //test
+                    unset($foobar  ); /* more comment test*/
+                    unset(  $foobar2);
+                ',
+            ),
+            // case that shows the depends on argument spaces after this fixer
+            array(
+                '<?php //2
+                    unset($foo,$bar,$foobar,$foobar2);/*1*/
+                    /*2*/
+                    //3
+                    /*4*/
+                ',
+                '<?php //2
+                    unset($foo);/*1*/
+                    unset($bar);/*2*/
+                    unset($foobar);//3
```

Comment:
we're missing a test case for when comments appear on their own line, or the unsets are separated by an extra line of whitespace